index.html

```diff
@@ -95,12 +95,12 @@ <h4>Wins</h4>
                     <div class="card_main_program">
                         <i class="fa-solid fa-user-plus"></i>
                         <h4>Deliver</h4>
-                        <p>With all the map of challenges, opportunities and process we activate execution!</p>
+                        <p>With the map of challenges, opportunities and process we activate execution!</p>
                     </div>
                 </div>
             </div>
             <button>Contact Us</button>
-            <p class="click_experience"><a href="">SEE THE WHOLE EXPERIENCE</a></p>
+            <p class="click_experience"><a href="">SEE THE EXPERIENCE</a></p>
```

Comment:
- [ ] please ensure that the different cards should have their background color different from the section background.
yours
![Screenshot (624)_LI](https://user-images.githubusercontent.com/65298048/203655779-cb504e2b-7f57-4fa9-a8ec-1cc39d92df3f.jpg)
behance
![Screenshot (625)](https://user-images.githubusercontent.com/65298048/203655848-c0f68041-ebac-4e9d-8e77-ff2d0d2d023a.png)